api/v1alpha1/nonadminbackup_types.go

```diff
@@ -51,9 +51,9 @@ type VeleroBackup struct {
 	// +optional
 	Status *velerov1.BackupStatus `json:"status,omitempty"`
 
-	// name references the Velero backup by it's name.
+	// nameuuid references the Velero Backup object by it's label containing same NameUUID.
 	// +optional
-	Name string `json:"name,omitempty"`
+	NameUUID string `json:"nameuuid,omitempty"`
```

Comment:
I do not like this change

This is still the Velero backup name

Is useful to user to know how the name is generated?

Reply:
The field indicates that it stores the VB name (ux) as well as the information that it is also the UUID (dev logic)